web/app.fluidity.money/app/util/chainUtils/links.ts

```diff
@@ -5,6 +5,12 @@ const getAddressExplorerLink = (chain: Chain, address: string) =>
     ? `https://etherscan.io/address/${address}`
     : `https://explorer.solana.com/address/${address}`;
 
+// Ethereum only
+const getBlockExplorerLink = (chain: Chain, block: number) =>
+  chain === "ethereum"
+    ? `https://etherscan.io/address/${block}`
+    : `https://explorer.solana.com/block/${block}`;
+
```

Comment:
Is there an actual need for this here ?, these links have been defined in `config.toml` file, instead of having an exclusive util function for this. you could pull it up from exactly where you are calling it on the rewards page i guess, using actual Loaders. kinda makes it cleaner.

Reply:
I'd say there's not much point in having it in the config file, since that's for backend stuff. We'll discuss later